RealmSwift/AsymmetricObject.swift

```diff
@@ -0,0 +1,100 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2022 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+import Foundation
+import Realm
+import Realm.Private
+
+/**
+ `AsymmetricObject` is a base class used to define asymmetric Realm objects.
+
+ Asymmetric objects can only be created using the `create(_ object:)`
+ function, and cannot be added, removed or queried.
+ When created, asymmetric objects will be synced unidirectionally to the MongoDB
+ database and cannot be accessed locally.
+
+ Incoming links from any asymmetric table are not allowed, meaning embedding
+ an asymmetric object within an `Object` will throw an error.
```

Comment:
"table" is an implementation detail and not a word that should appear in the docs.

I think this sentence is backwards; an incoming link from an asymmetric object would be an asymmetric object linking to another object, which is the opposite of embedding an asymmetric object within an `Object`.